worker/include/RTC/RtpPacket.hpp

```diff
@@ -473,9 +474,11 @@ namespace RTC
 			}
 			else if (HasOneByteExtensions())
 			{
-				auto it = this->mapOneByteExtensions.find(id);
+				if (id > 14)
+					return false;
 
-				return it != this->mapOneByteExtensions.end();
+				// `-1` because we have 14 elements total 0..=13 and `id` is in the range 1..=14
```

Comment:
```suggestion
				// `-1` because we have 14 elements total 0..=13 and `id` is in the range 1..=14.
```

BTW what does `0..=13` notation men?

Reply:
This is Rust syntax for ranges, `=` before second number means that that number is included in the range.

Reply:
👍🏼